##[group]==END_FILE==, 30.357948275588317, , , , 5764376, False, 
Requested labels: self-hosted, 450.691269551884, , , , 83912983, False, 
##[group]GITHUB_TOKEN Permissions, 183.78179553911903, , , , 2140690, False, 
##[group]Run actions/checkout@v4, 19.664257516077697, , , , 201692, False, 
##[group]Setting up auth, 10.048251694988345, , , , 71173, False, 
##[group]Fetching the repository, 51.53799609663868, , , , 462460, False, 
##[group]Installed versions, 10.82629426800652, , , , 125837, False, 
##[group]Run python -m pip install --upgrade pip, 138.9640400570703, , , , 20830917, True, ##[error]WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
##[group]Run cd docker, 29.929232249865112, , , , 12545677, False, 
##[group]==END_FILE==__dup_10, 951.6185962686828, , , , 112444338, False, 
##[group]GITHUB_TOKEN Permissions__dup_11, 6.392051753909241, , , , 2177350, False, 
##[group]Run actions/checkout@v4__dup_12, 3.740396072956274, , , , 198849, False, 
##[group]Setting up auth__dup_13, 1.9189068475651727, , , , 75158, False, 
##[group]Fetching the repository__dup_14, 7.8704626346088356, , , , 445348, False, 
##[group]Determining the checkout info, 2.0163244697392453, , , , 10213, False, 
##[group]Run python -m pip install --upgrade pip__dup_16, 63.16775033889588, , , , 10252562, True, ##[error]WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
##[group]Run cd docker__dup_17, 87.98604718059096, , , , 12229153, False, 
##[group]GITHUB_TOKEN Permissions__dup_18, 4.248389174748224, , , , 2199817, False, 
##[group]Run actions/checkout@v4__dup_19, 3.6663255196113433, , , , 189113, False, 
##[group]Removing previously created refs_ to avoid conflicts, 1.934902579618364, , , , 24988, False, 
##[group]Cleaning the repository, 2.0027291261601583, , , , 64040, False, 
##[group]Fetching the repository__dup_22, 11.031043828927716, , , , 450717, False, 
##[group]Installed versions__dup_23, 264.6863098170009, , , , 25318557, True, ##[error]WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
##[group]Run python -m pip install --upgrade pip__dup_24, 149.13485038096795, , , , 16930013, False, 
##[group]Run cd docker__dup_25, 176.2283114501021, , , , 17359277, False, 
##[group]GITHUB_TOKEN Permissions__dup_26, 73.38818919012738, , , , 6886914, False, 
##[group]Run actions/checkout@v4__dup_27, 8.555014045330706, , , , 243786, False, 
##[group]Getting Git version info, 7.624687398405049, , , , 344196, False, 
##[group]Cleaning the repository__dup_29, 1.709320037721288, , , , 81105, False, 
##[group]Setting up auth__dup_30, 1.801511316741427, , , , 82238, False, 
##[group]Fetching the repository__dup_31, 9.929469373908526, , , , 450014, False, 
##[group]Determining the checkout info__dup_32, 1.983735633541263, , , , 10894, False, 
##[group]Installed versions__dup_33, 1.857720530005088, , , , 8328, False, 
##[group]Run python -m pip install --upgrade pip__dup_34, 127.17814511381904, , , , 13914316, True, ##[error]WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
##[group]Run cd docker__dup_35, 171.3711733428785, , , , 11665349, False, 
